.github/workflows/benchmarks.yml

```diff
@@ -1,31 +1,31 @@
-name: benchmark
+name: Run Benchmarks
 
 on:
   workflow_dispatch:
 
 jobs:
   build:
     runs-on: ubuntu-latest
-    environment: benchmark
-    strategy:
-      matrix:
-        python-version: ['3.10', '3.11']
+
```

Comment:
Keep it a matrix with one item. It’s much easier to adopt more later

Reply:
Does running benchmarks in a matrix of different python versions make sense at all? To test if it works, we already have CI that runs in a matrix.

Reply:
Fair. Don’t do it for benchmarks. It’s just been common practice for teams I’m on to do it as general practice  don’t feel strongly here